transforms/create-element-to-jsx.js

```diff
@@ -0,0 +1,95 @@
+module.exports = function(file, api) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+
+  const convertObjectExpressionToJSXAttributes = (objectExpression) => {
+    if (!objectExpression.properties) {
+      return [];
+    }
+
+    const attributes = objectExpression.properties.map((property) => {
+
+      if (property.type === 'SpreadProperty') {
+        return j.jsxSpreadAttribute(property.argument);
+      }  else if (property.type === 'Property') {
+        const propertyValueType = property.value.type;
+
+        let value;
+        if (propertyValueType === 'Literal') {
+          value = j.literal(property.value.value);
+        } else if (propertyValueType === 'MemberExpression') {
+          value = j.jsxExpressionContainer(property.value);
+        }
+
+        return j.jsxAttribute(
+          j.jsxIdentifier(property.key.name),
+          value
+        );
+      }
+    });
+
+    return attributes;
+  };
+
+  const convertNodeToJSX = (node) => {
+    if (!node.value || node.value.type !== 'CallExpression') {
```

Comment:
I don't think you need this, right? Because you are returning undefined it would remove the node and the `find` call makes sure it is safe to call `convertNodeToJSX`. I'd prefer to put the guard around line #52